docs/src/main/asciidoc/openshift-s2i-guide.adoc

```diff
@@ -39,11 +39,38 @@ oc expose svc/quarkus-quickstart-native
 
 # Get the route URL
 export URL="http://$(oc get route | grep quarkus-quickstart-native | awk '{print $2}')"
+echo $URL
```

Comment:
What's the reason?

Reply:
@cescoffier just occurred to me that the text in the following line _("Your application is accessible at the printed URL.")_ does not really match the "code" (because the `curl` accesses the app, we don't actually print an URL) - see what I mean? It's of course really minor though, I can remove it again, or leave it, whatever you prefer.

Reply:
Good point!